install.sh

```diff
@@ -308,18 +327,12 @@ sudo ldconfig &> /tmp/crew_ldconfig || true
 echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
 
-echo_out "Set up and synchronize local package repo..."
-
 echo "export CREW_PREFIX=${CREW_PREFIX}" >> "${CREW_PREFIX}/etc/env.d/profile"
 
-# Install activesupport gem for ruby
-echo_info 'Installing essential ruby gems.'
-gem update -N --system
-gem install -N activesupport --conservative
-gem install -N concurrent-ruby --conservative
-
-# Since we downloaded the package repo, just update package compatibility information.
-crew update compatible
+echo_info 'Installing essential ruby gems...\n'
+BOOTSTRAP_GEMS='activesupport concurrent-ruby highline'
+# shellcheck disable=SC2086
+install_ruby_gem ${BOOTSTRAP_GEMS}
```

Comment:
Should these be packages and added to core.rb instead?

Reply:
Oh wait, they already are packages?  Why do we need to do this then?

Reply:
They are packages already using the `buildsystems/ruby` gem install system. This is just a way to force the install early without building from source, which using the ruby buildsystem forces, at least on i686.

Reply:
Maybe we need to rethink the ruby packages then?  It sort of makes sense to let ruby handle the package management heavy-lifting with a minimal crew wrapper.